native_client/definitions.mk

```diff
@@ -38,8 +40,48 @@ AS      := $(TOOLCHAIN)as
 CC      := $(TOOLCHAIN)gcc
 CXX     := $(TOOLCHAIN)c++
 LD      := $(TOOLCHAIN)ld
+LDD     := $(TOOLCHAIN)ldd $(TOOLCHAIN_LDD_OPTS)
 
+RPATH_PYTHON         := '-Wl,-rpath,\$$ORIGIN/lib/'
+RPATH_NODEJS         := '-Wl,-rpath,$$\$$ORIGIN/../../../'
 META_LD_LIBRARY_PATH := LD_LIBRARY_PATH
```

Comment:
You can now remove this variable and the `run` and `debug` targets in the Makefile.

Reply:
Since this is used to pickup the dependencies from TensorFlow's directory, I guess we do still need them :)